Lib/test/test_logging.py

```diff
@@ -3946,56 +3952,70 @@ def test_config_queue_handler(self):
             msg = str(ctx.exception)
             self.assertEqual(msg, "Unable to configure handler 'ah'")
 
+    def _apply_simple_queue_listener_configuration(self, qspec):
+        self.apply_config({
+            "version": 1,
+            "handlers": {
+                "queue_listener": {
+                    "class": "logging.handlers.QueueHandler",
+                    "queue": qspec,
+                },
+            },
+        })
+
     @threading_helper.requires_working_threading()
     @support.requires_subprocess()
     @patch("multiprocessing.Manager")
     def test_config_queue_handler_does_not_create_multiprocessing_manager(self, manager):
-        # gh-120868, gh-121723
-
-        from multiprocessing import Queue as MQ
-
-        q1 = {"()": "queue.Queue", "maxsize": -1}
-        q2 = MQ()
-        q3 = queue.Queue()
-        # CustomQueueFakeProtocol passes the checks but will not be usable
-        # since the signatures are incompatible. Checking the Queue API
-        # without testing the type of the actual queue is a trade-off
-        # between usability and the work we need to do in order to safely
-        # check that the queue object correctly implements the API.
-        q4 = CustomQueueFakeProtocol()
-
-        for qspec in (q1, q2, q3, q4):
-            self.apply_config(
-                {
-                    "version": 1,
-                    "handlers": {
-                        "queue_listener": {
-                            "class": "logging.handlers.QueueHandler",
-                            "queue": qspec,
-                        },
-                    },
-                }
-            )
-            manager.assert_not_called()
+        # gh-120868, gh-121723, gh-124653
+
+        for qspec in [
+            {"()": "queue.Queue", "maxsize": -1},
+            queue.Queue(),
+            # queue.SimpleQueue does not inherit from queue.Queue
+            queue.SimpleQueue(),
+            # CustomQueueFakeProtocol passes the checks but will not be usable
+            # since the signatures are incompatible. Checking the Queue API
+            # without testing the type of the actual queue is a trade-off
+            # between usability and the work we need to do in order to safely
+            # check that the queue object correctly implements the API.
+            CustomQueueFakeProtocol(),
+            MinimalQueueProtocol(),
+        ]:
+            with self.subTest(qspec=qspec):
+                self._apply_simple_queue_listener_configuration(qspec)
+                manager.assert_not_called()
 
     @patch("multiprocessing.Manager")
     def test_config_queue_handler_invalid_config_does_not_create_multiprocessing_manager(self, manager):
-        # gh-120868, gh-121723
+        # gh-120868, gh-121723, gh-124653
 
         for qspec in [object(), CustomQueueWrongProtocol()]:
```

Comment:
Here `multiprocessing.SimpleQueue` is not in the list ? Is it voluntary ?

Reply:
Sorry don't see below.

Reply:
Yes, it's being tested in a separate test (`test_config_reject_simple_queue_handler_multiprocessing_co`) so that I can also test all multiprocessing contexts separately.

Reply:
I should however update the comment since it's no more related to gh-124653.

Reply:
> Yes, it's being tested in a separate test (`test_config_reject_simple_queue_handler_multiprocessing_co`) so that I can also test all multiprocessing contexts separately.

Sorry for my late comment, but I am just wondering why you did not do the same thing (a specific test) with the `multiprocessing.Queue` use case.

Reply:
The `multiprocessing.Queue` case is handled in `test_multiprocessing_queues` (another test that was already there) and there are also other tests for the manager's queue proxy handled by `test_config_queue_handler_multiprocessing_context`. 

The file is a bit... disorganized and we should probably at one point merge some tests or clean them but that would be a huge diff. So we just leave it like this (I'm pretty sure that there are duplicated cases here and there).

Reply:
Thank you for your reply, Congratulations on this PR.